src/main/scala/me/rexim/morganey/ReplAutocompletion.scala

```diff
@@ -127,18 +130,34 @@ object ReplAutocompletion {
     }
   }
 
-  private object IsCommand {
+  private object SimpleCommand {
 
     def unapply(line: String): Option[List[String]] = {
       val potentialCommand = parseCommand(line).map(_._1)
       val matchingCommands = potentialCommand map { p =>
-        commands.keySet filter (_ startsWith p)
+        commands.values collect {
+          case c if c.name startsWith p =>
+            if (c.hasTermArgument) c.name + " " else c.name
```

Comment:
My goal with this code, was to autocomplete commands that have a term as an argument to ":command " (notice the space at the end). The space was, as far as I thought, necessary that the autocompletion for commands with arguments (`CommandWithArg`) can distinguish the command name and the arguments.

However, I was wrong. The space isn't needed.

I should have tested that earlier.